Clarify recommended UID for Unprivileged installation (#27924)
DataDog/documentation: content/en/containers/kubernetes/installation.md

        - <GROUP_ID>
{{< /highlight >}}

> - Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends to set 100 since Datadog Agent v7.48+][26].

Reviewer: ```suggestion
- Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends setting this value to 100 since Datadog Agent v7.48+][26].
```

Reviewer: Could we provide some kind of brief explanation for this here?

Author: Only a placeholder doesn't indicate what value to set. Since the documentation provides a recommended value, I'd like to include a link to it here.

https://github.com/DataDog/documentation/blob/63b3271a3abda3ba035d3773fba3dc07ec97c344/content/en/data_security/kubernetes.md?plain=1#L50-L51

Reviewer: Ah ok, I'll update my suggestion. By the way, this content is actually in a code block, so links need to be the complete URLs.

> - Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends setting this value to 100 since Datadog Agent v7.48+][26].

Reviewer: ```suggestion
- Replace `<USER_ID>` with the UID to run the Datadog Agent. Datadog recommends [setting this value to 100 since Datadog Agent v7.48+][1].

> - Replace `<GROUP_ID>` with the group ID that owns the Docker or containerd socket.

Reviewer: ```suggestion
- Replace `<GROUP_ID>` with the group ID that owns the Docker or containerd socket.

[1]: https://github.com/DataDog/documentation/blob/63b3271a3abda3ba035d3773fba3dc07ec97c344/content/en/data_security/kubernetes.md?plain=1#L50-L51
```

Author: I want to use this link instead of the GitHub url.
https://docs.datadoghq.com/data_security/kubernetes/#running-container-as-root-user

Reviewer: updated!

[23]: https://app.datadoghq.com/orchestration/resource/pod
[24]: /infrastructure/containers/orchestrator_explorer
[25]: /infrastructure/containers/kubernetes_resource_utilization

> [26]: /data_security/kubernetes

Reviewer: ```suggestion
```
